src/test/java/org/takes/facets/auth/social/PsGithubTest.java

```diff
@@ -148,4 +175,29 @@ private static void assertParam(final Request req,
             Matchers.equalTo(value)
         );
     }
+
+    /**
+     * An inner class for the Take implementation testing.
```

Comment:
@sebing in that case let's change this to something more detailed. How about
`A test Take that verifies if the request contains the access token and returns a JSON login response` ?